.buildkite/pipeline-for-tags.yml

```diff
@@ -33,6 +33,49 @@ steps:
    - buildkite-agent artifact download "out/*" . --step build-source-packages
    - nix develop .#autorelease -c ./scripts/publish-native-packages.sh out
```

Comment:
AFAIU this will still cause copr to try and build source packages for EPEL7 (and fail).

Can't this be avoided?

Reply:
It should be possible to do `--chroots fedora-{35,36}-{x86_64,aarch64}`, but this will require us to actually hardcode them:( 
Maybe there is a way to get all chroots for a given copr repo via API :thinking: